eslint/babel-eslint-plugin-development-internal/package.json

```diff
@@ -18,15 +18,12 @@
   "author": "Kai Cataldo <kai@kaicataldo.com>",
   "license": "MIT",
   "private": true,
-  "engines": {
```

Comment:
I removed the `engines` requirement here because its requirement can be implied from its peerDeps: `@babel/eslint-parser`.